### PR TITLE
eighthtask

### PR DESCRIPTION
**Pull request by YANGMAO-ZHANG**

### CODE REVIEW

**Comment by YANGMAO-ZHANG** [comment]
@Felixnoo

**Comment by Felixnoo** [comment]
#1854

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 1]
```diff
@@ -1,91 +1,91 @@
...
+description: '学习如何使用 webhook 触发 Jenkins 流水线'
+linkTitle: "使用 Webhook 触发流水线"
 weight: 11293
----
 
-If you create a Jenkinsfile-based pipeline from a remote code repository, you can configure a webhook in the remote repository so that the pipeline is automatically triggered when changes are made to the remote repository.
+---
 
-This tutorial demonstrates how to trigger a pipeline by using a webhook.
+如果从远程代码仓库创建 Jenkinsfile-based 流水线，则可以在远程仓库中配置 webhook，以便对远程仓库进行改变时，自动触发流水线。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:10]: `从` -> `通过`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
+本教程概述如何用 webhook 触发流水线。
 
-- You need to [enable the KubeSphere DevOps system](../../../pluggable-components/devops/).
-- You need to create a workspace, a DevOps project, and an account (`project-regular`). This account needs to be invited to the DevOps project and assigned the `operator` role. See [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/) if they are not ready.
+## 准备工作
 
-- You need to create a Jenkinsfile-based pipeline from a remote code repository. For more information, see [Create a Pipeline Using a Jenkinsfile](../create-a-pipeline-using-jenkinsfile/).
+- [启用 KubeSphere DevOps 系统](../../../pluggable-components/devops/)。
+- 创建一个企业空间， DevOps 工程和一个帐户（例如，`project-regular`）。`project-regular` 需要被邀请至 DevOps 工程中并赋予 `operator` 角色。有关更多信息，请参见[创建企业空间、项目、帐户和角色](../../../quick-start/create-workspace-and-project/)。
+- 在远程代码仓库，创建一个 Jenkinsfiel-based 流水线。有关更多信息，请参见[使用 Jenkinsfile 创建流水线](../create-a-pipeline-using-jenkinsfile/)
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:18]: `在远程代码仓库，创建一个 Jenkinsfiel-based 流水线` 
->
`通过远程代码仓库创建一个基于 Jenkinsfile 的流水线`

Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:18]: Add `。` at the end of this sentence.

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 3]
```diff
@@ -1,91 +1,91 @@
...
 
-## Configure a Webhook
+## 配置 Webhook
 
-### Get a webhook URL
+### 获得 webhook URL
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:22]: `获得` -> `获取`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 4]
```diff
@@ -1,91 +1,91 @@
...
 
-1. Log in to the KubeSphere web console as `project-regular`. Go to your DevOps project and click a pipeline (for example, `jenkins-in-scm`) to go to its details page.
+1. 使用 `project-regular` 帐户登录 Kubesphere web 控制台。转到 DevOps 工程，点击流水线（例如，`jenkins-in-scm`）以查看详情页面。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:24]: `web` -> `Web`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 5]
```diff
@@ -1,91 +1,91 @@
...
 
-2. Click **More** and select **Edit Config** in the drop-down list.
+2. 点击**更多**，在下拉菜单中选择**编辑配置文件**。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:26]: `编辑配置文件` -> `编辑配置`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 6]
```diff
@@ -1,91 +1,91 @@
...
 
-   ![edit-config](/images/docs/devops-user-guide/using-devops/pipeline-webhook/edit-config.png)
+   ![edit-config](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-webhook/edit-config.png)
 
-3. In the displayed dialog box, scroll down to **Webhook Push** to obtain the webhook push URL.
+3. 在出现的会话框中，滑动至 **Webhook Push** 以获得 webhook push URL。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:30]: `Webhook Push` -> `Webhook 推送`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 7]
```diff
@@ -1,91 +1,91 @@
...
 
-   ![webhook-push](/images/docs/devops-user-guide/using-devops/pipeline-webhook/webhook-push.png)
+   ![webhook-push](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-webhook/webhook-push.png)
 
-### Set a webhook in the GitHub repository
+### 在 GitHub 仓库中设置 webhook
 
-1. Log in to GitHub and go to your own repository `devops-java-sample`.
+1. 登录您的 GitHub，并转到 `devops-java-sample` 项目。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:36]: `项目` -> `仓库`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 8]
```diff
@@ -1,91 +1,91 @@
...
+   ![webhook-ready](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-webhook/webhook-ready.png)
 
-## Trigger the Pipeline by Using the Webhook
+## 使用 Webhook 触发流水线
 
-### Submit a pull request to the repository
+### 提交拉取请求到仓库
 
-1. On the **Code** page of your own repository, click **master** and then select **sonarqube**.
+1. 在您仓库的**代码**页面，点击 **master** 然后选择 **sonarqube**。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:54]: **代码** -> **Code**
Let's use the UI text here.

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 9]
```diff
@@ -1,91 +1,91 @@
...
+### 检查 webhook 交付
 
-1. On the **Webhooks** page of your own repository, click the webhook.
+1. 查看在您仓库的 **Webhooks** 页面，点击 webhook。
 
-   ![webhook-ready](/images/docs/devops-user-guide/using-devops/pipeline-webhook/webhook-ready.png)
+   ![webhook-ready](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-webhook/webhook-ready.png)
 
-2. Click **Recent Deliveries** and click a specific delivery record to view its details.
+2. 点击 **Recent Deliveries**，然后点击 specific delivery record 查看详情。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:74]: `specific delivery record` -> `一个具体交付记录`

File: content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md [hunk 10]
```diff
@@ -1,91 +1,91 @@
...
 
-   ![delivery-detail](/images/docs/devops-user-guide/using-devops/pipeline-webhook/delivery-detail.png)
+   ![delivery-detail](/images/docs/zh-cn/devops-user-guide/use-devops/pipeline-webhook/delivery-detail.png)
 
-### Check the pipeline
+### 检查流水线
 
-1. Log in to the KubeSphere web console as `project-regular`. Go to your DevOps project and click the pipeline.
+1. 使用 `project-regular` 帐户登录 Kubesphere web 控制台。转到 DevOps 工程，点击流水线。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/how-to-use/pipeline-webhook.md:80]: `web` -> `Web`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please take a look at the comments, thanks.

**Comment by Felixnoo** [comment]
/assign @Patrick-LuoYu 
Hi, this PR is submitted by the OSSC student. I reviewed it already. Please take a look at it when you are available, thanks.

**Pull request merged**